assets/src/components/nav/navMenu.tsx

```diff
@@ -22,10 +18,12 @@ const NavMenu: React.FC<Props> = ({ mobileMenuIsOpen, toggleMobileMenu }) => {
     <>
       <div
         data-testid="nav-menu"
-        className={
-          "c-nav-menu" +
-          (mobileMenuIsOpen ? " c-nav-menu--open" : " c-nav-menu--closed")
-        }
+        className={joinClasses([
+          "c-nav-menu",
```

Comment:
Non-blocking nit: I couldn't find `c-nav-menu` in the CSS, so I think that means we can get rid of it here.

Reply:
I'm not really sure what BEMs guidance would be here, technically this is here in case any of the other BEM classes need to reference the component root. My instinct is not to remove it as that would leave all the related BEM styles "dangling"

Reply:
I see I see. That makes sense. That's another thing that, if needed, we can revisit.